src/components/Modals/CreateItemModal/CreateItemModal.tsx

```diff
@@ -96,14 +96,46 @@ export default class CreateItemModal extends React.PureComponent<Props, State> {
     return state
   }
 
+  /**
+   * Creates a new contents record with the names of the contents blobs record prefixed.
+   * The names need to be prefixed so they won't collide with other
+   * pre-uploaded models. The name of the content is the name of the uploaded file.
+   *
+   * @param bodyShape - The body shaped used to prefix the content names.
+   * @param contents - The contents which keys are going to be prefixed.
+   */
+  prefixContents(bodyShape: BodyShapeType, contents: Record<string, Blob>): Record<string, Blob> {
+    return Object.keys(contents).reduce((newContents: Record<string, Blob>, key: string) => {
+      newContents[`${bodyShape}:${key}`] = contents[key]
```

Comment:
let's use a `/` instead of a `:` to namespace these, so it's a valid path (would be handy if we needed to convert the contents into zip files, as we do with scenes or assetpacks.